mainpage.dox

```diff
@@ -574,6 +584,123 @@ janus.attach(
  *
  */
 
+/*!\page js-dependencies Working with custom \c janus.js dependencies
+ *
+ * Certain dependencies of \c janus.js may be passed during library initialization as
+ * a property list containing the following keys:
+ *
+ * -# \c newWebSocket: a function which given WebSockets server and protocol arguments
+ * should return a new WebSocket (or something that acts like it)
+ * -# \c webRTCAdapter: an \c adapter object such as provided by the
+ * <a href="https://github.com/webrtc/adapter">webrtc-adapter</a> library
+ * -# \c isArray: a function which tests if a given argument is a JavaScript array
+ * -# \c checkJanusExtension: a function which tests if the Janus Screensharing extension
+ * for Chrome is installed/available
+ * -# \c httpAPICall: a function which given an url and options argument performs an
+ * HTTP API request to the Janus Gateway. This function is not as straightforward to implement,
+ * see \ref js-http-apicall the documentation on \c httpAPICall below for details.
```

Comment:
See above on the usage of \ref, I guess here you can replace the existing line:

    see \ref js-http-apicall the documentation on \c httpAPICall below...

with this:

    see the documentation on \ref js-http-apicall below

as the `js-http-apicall` already has an `httpAPICall` label which would be inserted inline.